recipes/fa2/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "fa2" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fa2-{{ version }}.tar.gz
```

Taken from [the example]( https://github.com/conda-forge/staged-recipes/blob/2efe131304e7f0ca00bad5168cabf657086da12b/recipes/example/meta.yaml#L14 ).

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

---

Change completed.